app/com/gu/viewer/views/viewer.scala.html

```diff
@@ -33,9 +33,9 @@ <h1 class="top-toolbar__page-title">
                     <li data-switchmode="reader" class="top-toolbar__button is-always-visible"><i class="top-toolbar__icon--copycheck"></i>Copy taste<!--</li>-->
                     @if(previewEnv == "preview") {
                         <li class="top-toolbar__button--mobilecheck" data-toggledesktop="true"><i class="top-toolbar__icon--checkbox"></i>Mobile checked & verified<!--</li>-->
-                        <li data-switchmode="desktop" class="top-toolbar__button"><i class="top-toolbar__icon--desktop"></i>Desktop<!--</li>-->
+                        <li data-switchmode="desktop" class="top-toolbar__button show-desktop-enabled"><i class="top-toolbar__icon--desktop"></i>Desktop<!--</li>-->
```

Comment:
Is `show-desktop-enabled` a variant or modifier? If so, it could be `top-toolbar__button--desktop-enabled` or something similar.